CMakeLists.txt

```diff
@@ -572,9 +570,7 @@ if(NOT AVIF_CODEC_AOM_ENABLED
     message(WARNING "libavif: No decoding library is enabled.")
 endif()
 
-if(AVIF_LIB_USE_CXX OR AVIF_BUILD_APPS OR (AVIF_BUILD_TESTS AND (AVIF_ENABLE_FUZZTEST
-                                                                                                         OR AVIF_ENABLE_GTEST))
-)
+if(AVIF_LIB_USE_CXX OR AVIF_BUILD_APPS OR (AVIF_BUILD_TESTS AND (AVIF_ENABLE_FUZZTEST OR AVIF_GTEST)))
```

Comment:
Do you think `AVIF_ENABLE_FUZZTEST` can receive the same treatment?

Reply:
Yes I'm working on it.

Reply:
Indeed, we talked about it but I wanted to add more stuff at the same time.
I fixed it in https://github.com/AOMediaCodec/libavif/pull/2529 (and fuzztest is now downloadable from CMake just like any other dependency)